<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureDialog</class>
 <widget class="QDialog" name="CaptureDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Capture</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="camera_layout">
     <item>
      <widget class="QGroupBox" name="left_camera_group">
       <property name="title">
        <string>Left Camera</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="left_camera_combo">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ImageLabel" name="left_camera_image" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="right_camera_group">
       <property name="title">
        <string>Right Camera</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="right_camera_combo">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ImageLabel" name="right_camera_image" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grub_group">
     <property name="title">
      <string>Grub</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0" columnstretch="0,2,2,2,0">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="img_count_label">
        <property name="text">
         <string>Count</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mode_label">
        <property name="text">
         <string>Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="mode_auto_button">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="corner_count_label">
        <property name="toolTip">
         <string>Interior number of corners</string>
        </property>
        <property name="text">
         <string>Corners</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="max_images_spin">
        <property name="minimum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="corner_count_x_spin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="output_dir_label">
        <property name="text">
         <string>Output folder:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="output_dir_line"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="output_dir_button">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QCheckBox" name="auto_detect_corners_check">
        <property name="text">
         <string>Auto detect corners</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QSpinBox" name="corner_count_y_spin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QRadioButton" name="mode_hand_button">
        <property name="text">
         <string>Hand</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="current_message_label">
     <property name="text">
      <string>message</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="progress_label">
         <property name="text">
          <string>Progress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="capture_button">
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_cancel_button">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QWidget</extends>
   <header>ImageLabel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
